scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -39,6 +39,7 @@ import org.apache.spark.expose.Logging
 import org.apache.spark.ml.Model
 import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
 import org.apache.spark.sql.types._
+import ai.h2o.sparkling.utils.DataFrameSerializationWrappers._
```

Comment:
Is the additional import needed here?

Reply:
yes, [here](https://github.com/h2oai/sparkling-water/pull/2659/files/c4b15fc10c7d22afbbb0d465a0d6a8dc8d0da71d#diff-5b64d0b9c48d4b1b735251846a288e0c159484ca06de444229c98853b3ba2723L200) for example. `$(crossValidationMetricsSummary)` returns `DataFrameSerializationWrapper` and imported implicit method `toDataFrame` will look after the conversion to `DataFrame`.